torchci/lib/JobClassifierUtil.ts

```diff
@@ -120,9 +120,12 @@ export function isFailure(conclusion?: string): boolean {
       return false;
   }
 }
-export function getConclusionChar(conclusion?: string): string {
+export function getConclusionChar(conclusion?: string, failedPreviousRun?: boolean): string {
   switch (conclusion) {
     case JobStatus.Success:
+      if (failedPreviousRun) {
+        return "F"
+      }
       return "O";
     case JobStatus.Failure:
```

Comment:
Could we also show failed retry here? I dont mind reusing the same F char with different color code, but we can also think of something else, like a red number of retries?

Reply:
I think I'm going to leave it as is for now, I'm already confusing myself with following all the branches for what is what color and what letter things are